### PR TITLE
Add yqla token file in container

### PR DESCRIPTION
**Pull request by Krisha11**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: pkg/components/yql_agent.go [hunk 1]
```diff
@@ -141,8 +141,10 @@ func (yqla *yqlAgent) doSync(ctx context.Context, dry bool) (ComponentStatus, er
 
 	if yqla.NeedSync() {
 		if !dry {
+			token, _ := yqla.secret.GetValue(consts.TokenSecretKey)
 			ss := yqla.server.buildStatefulSet()
 			container := &ss.Spec.Template.Spec.Containers[0]
+			container.Command = []string{"sh", "-c", fmt.Sprintf("echo -n %s > /usr/yql_agent_token; %s", token, strings.Join(container.Command, " "))}
```
Comment (gritukan) [inline pkg/components/yql_agent.go:147]: Won't token be shown in `ps` or pod description in this case?

I suggest to use environment variable. Looks like it is already passed, so `echo $SECRET >/usr/yql_agent_token` should be sufficient.

Comment (gritukan) [inline pkg/components/yql_agent.go:147]: Shouldn't we use `YTTokenPath` instead of hardcoded `/usr/yql_agent_token` here?

Comment (Krisha11) [reply pkg/components/yql_agent.go]: It is not very easy to access the config here. It has already been serialized by this point. So I put token path in consts

**Comment by l0kix2** [comment]
@gritukan do you have any notes here or we can merge it?

**Review by gritukan — APPROVED** [review]

**Pull request merged**